src/content/docs/guides/json/2-json_writing.mdx

```diff
@@ -31,6 +31,17 @@ In SplashKit, you can programmatically create JSON objects and arrays, which the
   ```
 
   </TabItem>
+
+  <TabItem label="C#">
+
+  ```csharp
+  var newGameData = Json.Create();
```

Comment:
These functions do not exist. 

See below:

Json newGameData = SplashKit.CreateJson();
SplashKit.JsonSetString(newGameData, "gameTitle", "My New Game");
SplashKit.JsonSetBool(newGameData, "fullScreenMode", false);
SplashKit.JsonSetNumber(newGameData, "numPlayers", 1);